CONTRIBUTING.md

```diff
@@ -38,37 +52,45 @@ To submit a feature request, follow these steps:
 
 The best way to see a feature added, however, is to submit a pull request.
 
-## Pull Requests
-
-* Before creating your pull request, it's usually worth asking whether the code you're planning on writing will be considered for merging. You can do this by [opening an issue][1] and asking. It may also help give the maintainers context for when the time comes to review your code.
+#### Pull Requests
 
-* Ensure your [commit messages are well-written][2]. This can double as your pull request message, so it pays to take the time to write a clear message.
+Before creating your pull request, it's usually worth asking whether the code you're planning on writing will be considered for merging. You can do this by [opening an issue][1] and asking. It may also help give the maintainers context for when the time comes to review your code.
 
-* Add tests for your feature. You should be able to [look at other tests][4] for examples. If you're unsure, don't hesitate to [open an issue][1] and ask!
+Ensure that your [commit messages are well-written][2]. This can double as your pull request message, so it pays to take the time to write a clear message.
 
-* Submit your pull request!
+Additionally, make sure that you have written unit tests for your changes. If you're unsure as to what to test, don't hesitate to [open an issue][1] and ask!
```

Comment:
Actually we don't need unit tests for this repo and this is just a specification repo so we can remove it

Reply:
Removed in next commit.